### DISCUSSION TITLE
Mapping of metastases to Metastasis class in Cancer Modifier vocabulary

### DISCUSSION THREAD

**Discussion by peterprinsen-iknl** [OHDSI/OncologyWG#500] [General]
At the Netherlands Cancer Registry (NCR) we register metastases by their ICD-O-3 topography and the laterality (left/right/bilateral/central/unknown). The algorithm for adding this to our OMOP-CDM works as follows:
* Take the ICD-O-3 topography CXY.Z of the metastasis and create the code 8000/6-CXY.Z (for example 8000/6-C22.0).
* Find this concept code in the vocabulary and look up the standard concept (in this case 36770544 - Metastasis to liver).
* Add this concept to the MEASUREMENT table (and link the measurement to the primary cancer in the usual way, i.e. via measurement_event_id and meas_event_field_concept_id).
* Map the laterality to a standard concept in the Cancer Modifier vocabulary (Topography class: left -> Left (36770232), right -> Right (36770058), bilateral -> Bilateral (36770109), central -> Central (36769018), unknown -> 0).
* Add this laterality concept to the MEASUREMENT table and link it to the metastasis in the MEASUREMENT table to which it pertains (via measurement_event_id and meas_event_field_concept_id).

This works for a lot of the topographies but not all (if it does not work it is because 8000/6-CXY.Z does not exist). We therefor created a mapping table for the remaining topographies (see [metastases](https://github.com/OHDSI/OncologyWG/files/10969538/metastases.for.discussion.xlsx)). For most topographies we can find a satisfactory mapping (green color coding), for some we cannot (orange rows).

My questions are:
* Do other data sources register metastases the same way and if so, did they run into the same issues when trying to map them?
* Does anyone have better mappings for the orange rows?
* Would it make sense to add additional metastasis concepts to the Cancer Modifier vocabulary so that the orange rows in the file can be mapped to a satisfactory concept? Caveat here is that "not satisfactory" is my evaluation. It is not based on a use case so adding these concepts to the vocabulary may solve a non-existent problem.

**Comment by cgreich** [comment]
@peterprinsen-iknl:

You guys use ICDO topology for metastases? It's meant for the primaries, but I guess you can do that. 

But: Do you really have in the data e.g. a metastasis to "Overlapping lesion of lip, oral cavity and pharynx"? That would be extremely odd. It's a typical situation a primary presents. Remember: primaries are derived from cells typically replicating heavily (e.g. epithelial cells of the mucosa of inner organs), metastasis happen when a malignant cell travels through the blood stream and gets stuck somewhere else in the body. So, before stressing over the orange lines I would actually check out which ones you really need.

Overall: As discussed before - the exact location of a metastasis is false precision for almost all imaginable use cases. It really doesn't matter to prognosis or treatment into which bone or into which soft tissue a primary metastasizes. There is some difference between lung, brain, liver and bone, but that's marginal. Patients may get a palliative surgery if the met creates trouble, but otherwise the treatment is decided on functional impairment in daily life, i.e. how fit the patient is. The prognosis is rarely affected.
